test/lib/ufe/testDeleteCmd.py

```diff
@@ -310,10 +421,240 @@ def testDeleteArgs(self):
         propsChildrenNames = childrenNames(propsChildren)
 
         self.assertNotIn(sphereShapeName, sphereChildrenNames)
-        self.assertNotIn(ball35Name, propsChildrenNames)
-        self.assertNotIn(ball34Name, propsChildrenNames)
+        self.assertIn(ball35Name, propsChildrenNames)
+        self.assertIn(ball34Name, propsChildrenNames)
         self.assertFalse(cmds.objExists("|pSphere1|pSphereShape1"))
+        
+        cmds.undo()
+
+    @unittest.skipUnless(ufeUtils.ufeFeatureSetVersion() >= 2, 'testDeleteProxyShape only available in UFE v2 or greater.') 
+    def testDeleteProxyShape(self):
+        '''Delete proxy shape.'''
+
+        # Create our UFE notification observer
+        ufeObs = TestObserver()
+        ufe.Scene.addObserver(ufeObs)
+
+        mayaSegment = mayaUtils.createUfePathSegment(
+            "|transform1|proxyShape1")
+        usdSegment = usdUtils.createUfePathSegment("/Room_set/Props/Ball_35")
+
+        ball35Path = ufe.Path([mayaSegment, usdSegment])
+        ball35PathStr = ','.join(
+            [str(segment) for segment in ball35Path.segments])
+
+        self.assertEqual(len(cmds.ls('proxyShape1')), 1)
+
+        # delete the proxy shape node itself. 
+        cmds.delete('|transform1|proxyShape1')
+
+        self.assertEqual(ufeObs.nbDeleteNotif(), 1)
+        
+        #restore
+        cmds.undo()
+
+        ufeObs.reset()
+        self.assertEqual(len(cmds.ls('proxyShape1')), 1)
```

Comment:
I would remove this test, because it's essentially testing that you can delete a Maya node, which Maya is already supposed to test.

Reply:
removed